db/seeds.rb

```diff
@@ -1,3 +1,7 @@
 seeder = DataSeeder.new
 seeder.generate_dump_types
 seeder.generate_dump_file_types
+
+10.times do |i|
+  Event.find_or_create_by(id: i+1, start: "200#{i}-10-10", finish: "200#{i}-10-11", success: true)
```

Comment:
I like the year interpolation!

Reply:
It looks like there are some failing tests.  I think because you are specifying ids here, the database doesn't think to increment the id sequence, so there are failures when it attempts to create a second row with the same ID (e.g. 1).

If you don't specify an id in this line, and just let the database keep track of assigning IDs, it should fix it.

Reply:
Thanks for the feedback!